modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceQuickAddToCart.testcase

```diff
@@ -551,49 +738,47 @@ definition {
 
 		}
 
-		task ("And searches for an invalid sku on the quick add-to-cart") {
+		task ("And searches for MIN93015 on the quick add-to-cart") {
 			Type(
 				locator1 = "Publications#KEYWORD_TEXTFIELD",
-				value1 = "MIN99999");
+				value1 = "MIN93015");
 		}
 
-		task ("Then no results are shown") {
+		task ("Then the result is not shown") {
 			AssertElementNotPresent(
-				key_productSku = "MIN",
+				key_productSku = "MIN93015",
 				locator1 = "CommerceAccelerators#QUICK_ADD_TO_CART_SEARCH_RESULT");
 		}
 
-		task ("And the search bar color is changed to red") {
-			KeyPress(
-				locator1 = "Publications#KEYWORD_TEXTFIELD",
-				value1 = "\ENTER");
-
-			/*There's the bug COMMERCE-10474 when pressing the enter keyboard button on the searchbar
-			but I'm using it in this test anyway because in this particular case the bug doesn't affect it*/
+		task ("When the admin logs in") {
+			User.logoutPG();
 
-			AssertElementPresent(locator1 = "TextInput#ALERT");
+			CommerceLogin.miniumLogin(
+				password = "test",
+				urlAppend = "web/minium",
+				userEmailAddress = "test@liferay.com");
 		}
-	}
 
-	@description = "This is a test for COMMERCE-10390. Buyer user can quick add to cart and complete a checkout"
-	@priority = "4"
-	test CanQuickAddToCartAndCheckout {
-		property custom.properties = "jsonws.web.service.paths.excludes=${line.separator}company.security.strangers.verify=false";
-		property portal.acceptance = "false";
+		task ("And the SKU is configured as purchasable") {
+			CommerceNavigator.gotoPortlet(
+				category = "Product Management",
+				portlet = "Products");
 
-		task ("Given an account with a buyer user is created") {
-			CommerceEntry.addAccountEntryUser(
-				accountName = "Commerce Account",
-				accountType = "Business",
-				agreeToTermsAndAnswerReminderQuery = "true",
-				createNewAccount = "true",
-				requireReset = "false",
-				userEmailAddress = "buyer@liferay.com",
-				userFirstName = "Buyer",
-				userLastName = "User",
-				userRole = "Buyer",
-				userScreenName = "buyeruser",
-				userSiteMembership = "Minium");
+			CommerceProducts.searchAndGoToProductEntry(entryProductName = "ABS Sensor");
+
+			CommerceEntry.gotoMenuTab(menuTab = "SKUs");
+
+			CommerceNavigator.gotoEntry(entryName = "MIN93015");
+
+			SelectFrame.selectFrameNoLoading(locator1 = "CommerceEntry#IFRAME_SIDE_PANEL");
+
+			Check.checkToggleSwitch(
+				key_toggleSwitchLabel = "Purchasable",
+				locator1 = "ToggleSwitch#ANY_TOGGLE_SWITCH");
+
+			Button.clickPublish();
+
+			CommerceUtilities.checkSuccessAlertAppearsAndDisappears();
```

Comment:
An API can be used for this. Check _patchCommerceSku macro and PATCH
/v1.0/skus/{id}